ejb-timer/.github/quickstart_ejb_timer_ci.yml

```diff
@@ -0,0 +1,21 @@
+name: WildFly EJB Timer Quickstart CI
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths:
+      - <quickstart-path>/**'
+      - .github/workflows/quickstart_ci.yml
+
+jobs:
+  call-quickstart_ci:
+    uses: ./.github/workflows/quickstart_ci.yml
+    with:
+      QUICKSTART_PATH: ejb-timer
+      DEPLOYMENT_DIR: webapp
+      TEST_PROVISIONED_SERVER: true
+      TEST_BOOTABLE_JAR: true
+      TEST_OPENSHIFT: true
+      SERVER_PROVISIONING_SERVER_HOST: https://localhost:8443
```

Comment:
this QS uses the standard server host, thus please remove this line

Reply:
fixed